ios/Classes/SwiftAmplitudeFlutterPlugin.swift

```diff
@@ -57,6 +57,10 @@ import Amplitude
                     let libraryVersion = args["libraryVersion"] as! String
                     Amplitude.instance(withName: instanceName).libraryVersion = libraryVersion
                     result(true)
+                case "setEventUploadThreshold":
+                    let eventUploadThreshold = args["eventUploadThreshold"] as! Int32
```

Comment:
I think we should use `Int` here

Reply:
And then make the conversion explicit on the next line with:
```
Amplitude.instance(withName: instanceName).eventUploadThreshold = Int32(eventUploadThreshold)
```
?

Reply:
Oh you're right. I thought `eventUploadThreshold` is a NSInteger. Nvm then, this looks good!